cypress-prioritization-and-triage.md

```diff
@@ -0,0 +1,123 @@
+
+
+## Cypress Priorities
+
+Cypress is constantly looking to improve and grow our testing platform to meet the ever-growing needs of the testing world. Like any software platform, we must balance research, development, and maintenance against real-world resource constraints. We feel that transparency about our future plans will help us grow a stronger relationship with end users who understand that all choices involve trade-offs.
+
+## Cypress Roadmap
+
+Our [high level roadmap](https://github.com/orgs/cypress-io/projects/13/views/1) shows high-level projects at Cypress. 
+
+The board flows from left to right as a ticket moves from the ideation and feasibility phase all the way to General Availability (GA) release. 
+
+Here is a guide to what each column on the board represents:
+
+| Column Name                           | Description            | 
+ | :------------------------------------ | :---------------------- | 
+ | **Under Consideration**               | These tickets are currently being discussed for future work. We are investigating the complexity of the change, the amount of resources necessary to complete it, and how it aligns with other goals for the platform. It is important to note that **not all** items in this column make their way into our scheduled work. It is possible that, upon investigation, a ticket falls in priority due to other goals or constraints.               | 
+| **Planned** |  These are the tickets we are planning to work on, but the work has not yet started. |
+| **In Progress** | These tickets are actively being worked on. |
+| **Experimental** | If a ticket involves a substantial refactoring of the codebase, is large in scope or is a complex feature, we often release them in a turned-off state. End users can opt in to experiment with these changes via feature flags. Work is still ongoing, but we feel it is important to get this feature in users' hands. Feedback is always welcome, but it is especially helpful during this phase of the development cycle. Not all issues are released under the experimental flag before GA. |
+| **Released** | Once an issue has been fully implemented, tested, and bundled into an official release it moves to this column. Features in this column are no longer experimental, feature flags are removed and default behaviors are implemented. |
+
+## Cypress Issue Triage
+
+Whenever an issue is created in the [Cypres repo](https://github.com/cypress-io/cypress/) it is added to the [Cypress App Triage Board](https://github.com/orgs/cypress-io/projects/9/views/1). This board represents all issues we are actively working to investigate and reproduce before routing them to the appropriate team for [prioritization](#how-we-prioritize-issues). Prioritization does not necessarily mean that a ticket will be worked on - priority scoring simply helps us create a relative ordering of potential work that is ranked according to standards we have defined as most important to Cypress and the community.
```

Comment:
'ticket' and 'issue' are both used here. Suggest using 'issue' for this section.